src/openrct2/ride/Ride.cpp

```diff
@@ -2126,14 +2126,14 @@ void Ride::UpdateChairlift()
     if (old_chairlift_bullwheel_rotation == speed / 8)
         return;
 
-    x = chairlift_bullwheel_location[0].x * 32;
-    y = chairlift_bullwheel_location[0].y * 32;
-    z = chairlift_bullwheel_z[0] * 8;
+    x = ChairliftBullwheelLocation[0].x * 32;
```

Comment:
Use `ToCoordsXYZ`.

Reply:
I had kind of left this out as it would cause collision with the map invalidate pr but I've made this change now anyway.